docs/workflow/blocks/actions.rst

```diff
@@ -152,4 +152,9 @@ After the block as loaded, the `enabled` property will be set on the button:
       ]
     }
 
-Buttons do not work for external links. Consider the :doc:`Template Block <template>` for external links.
\ No newline at end of file
+Buttons do not work for external links. Consider the :doc:`Template Block <template>` for external links.
+
+See also
```

Comment:
You're right, I should merge these sections better. I should also get rid of "Buttons do not work for external links". I only edited the above line because I saw a syntax warning somewhere. I didn't look at it enough! So good catch, thank you.

Reply:
To be clear, the Link Action Block does not support external links either (at least not right now), but I will clarify this. I do think it should concisely explain how they are relevant though.

Reply:
New text looks great to me! +1